library/Icinga/Web/Menu.php

```diff
@@ -54,6 +54,14 @@ public function init()
                     'label'       => t('Announcements'),
                     'url'         => 'announcements',
                     'priority'    => 710
+                ],
+                    'sessions'    => [
```

Comment:
That's still wrong :laughing: 

That's correct:

```suggestion
                ],
                'sessions' => [
```